src/skin/colorschemeparser.cpp

```diff
@@ -39,6 +39,11 @@ void ColorSchemeParser::setupLegacyColorSchemes(QDomElement docElem,
             }
         }
 
+        if (!bSelectedColorSchemeFound) {
+            // If we didn't find a matching color scheme, accept the first one in the file
+            bSelectedColorSchemeFound = true;
+        }
+
         if (bSelectedColorSchemeFound) {
```

Comment:
Can you remove this if condition? I don't think its possible to have this bool evaluate to false anymore.

Reply:
ping

Reply:
will take care of this soonish.

Reply:
Don't hurry, just wasn't sure if you've seen it.

Reply:
Yes, the if condition can be removed.
But now that I'm at it I'll also check what situations a user might run into, for example with a corrupted downloaded custom skin, and if that would load a valid skin nonetheless, or at least if removing the custom skin.

Reply:
I leave the condition and reset to the first scheme if there was no scheme name match, because after the while loop `schemeNode` is probably not the first scheme anymore.
I omit any checks for valid chuld nodes and add a hint to the color scheme documentation instead.